### PR TITLE
Update `min_position_notional_liquidated` to 1000 USDC

### PR DESCRIPTION
**Pull request by lucas-dydx**
Update `min_position_notional_liquidated` to 1000 USDC.

[Slack thread for context](https://dydx-team.slack.com/archives/C04BNGM82HM/p1697559168985669).

### Changelist
[Describe or list the changes made in this PR]

### Test Plan
[Describe how this PR was tested (if applicable)]

### Author/Reviewer Checklist
- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
- [ ] If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
- [ ] If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
- [ ] Manually add any of the following labels: `refactor`, `chore`, `bug`.

### CODE REVIEW

**Review by IvoCrnkovic — APPROVED** [review]
`genesis.sh` looks good to me. 

hard to read the `sample_pregenesis.json` diffs

**Review by matthewdowney — APPROVED** [review]
lgtm!

File: protocol/scripts/genesis/sample_pregenesis.json
Comment (jayy04) [inline protocol/scripts/genesis/sample_pregenesis.json:1]: for my understanding - what is this used for and how did we generate this?

**Review by jayy04 — APPROVED** [review]

File: protocol/scripts/genesis/sample_pregenesis.json
Comment (teddyding) [reply protocol/scripts/genesis/sample_pregenesis.json:1]: This is for human preview only. We rely on this file as the source of truth for what the default genesis parameter is for the dYdX chain

**Comment by matthewdowney** [comment]
@teddyding how'd you regenerate `sample_pregenesis.json` while preserving the key order?

**Comment by teddyding** [comment]
@matthewdowney I added a PR to also sort the output json in `make update-sample-pregenesis`, which should help!

**Pull request merged**